analyzers/tests/SonarAnalyzer.UnitTest/TestCases/DisposeFromDispose.CSharp10.cs

```diff
@@ -19,10 +19,19 @@ record struct R1 : IDisposable
     public void Dispose() { }
 }
 
+record struct R2 : IDisposable
+{
+    private FileStream Fs { get; set; }
+
+    public void OpenResource(string path) => this.Fs = new FileStream(path, FileMode.Open);
+    public void CloseResource() => this.Fs.Close();
+    public void CleanUp() => this.Fs.Dispose(); // Compliant. Properties are not not considered.
+    public void Dispose() { }
+}
 
 public record struct PositionalRecordStruct(FileStream fs) : IDisposable
 {
     public void CloseResource() => this.fs.Close();
-    public void CleanUp() => this.fs.Dispose(); // FN
+    public void CleanUp() => this.fs.Dispose(); // Compliant. FileStream fs is a public property
```

Comment:
`fs` is a `public` property. The [Rspec](https://jira.sonarsource.com/browse/RSPEC-2952) isn't clear about what kind of fields/properties should be considered by the analyzer but skipping `public` members seems to be the right choice.